solidus_dev_support.gemspec

```diff
@@ -40,7 +40,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'puma', '~> 4.3'
   spec.add_dependency 'rspec_junit_formatter'
   spec.add_dependency 'rspec-rails', '~> 4.0.0.beta3'
-  spec.add_dependency 'rubocop', '~> 0.87.1'
+  spec.add_dependency 'rubocop', '~> 0.90.0'
```

Comment:
I think we should relax rubocop version, so you can update in your project if needed.

Something like:

```suggestion
  spec.add_dependency 'rubocop', '~> 0.90'
```

Btw, current rubocop version is `0.91` now https://rubygems.org/gems/rubocop

Reply:
@ccarruitero Thanks! I totally agree, updated that just now.